Jenkinsfile

```diff
@@ -5,7 +5,7 @@ buildPlugin(
   // cannot use this with Docker tests
   useContainerAgent: false,
   configurations: [
-    [ platform: "linux", jdk: "21" ],
-    [ platform: "windows", jdk: "17" ]
+    [ platform: "linux", jdk: 21 ],
+    [ platform: "windows", jdk: 17 ]
```

Comment:
I know that the README of the jenkinsci pipeline library says exactly that, but jdk versions are String:
https://github.com/jenkins-infra/pipeline-library/blob/master/vars/buildPlugin.groovy#L33

Reply:
As for the content of the file, I was (blindly?) following the [official archetype](https://github.com/jenkinsci/archetypes/blob/master/common-files/Jenkinsfile). 🤷